os/mm/umm_heap/umm_zalloc.c

```diff
@@ -137,7 +141,7 @@ static void *heap_zalloc(size_t size, int s, int e, size_t retaddr)
 		}
```

Comment:
Need to check line 133.
heap_idx < e;
->
heap_idx <= e;

Reply:
Modified. Thanks.